OctopusDSC/DSCResources/cTentacleAgent/cTentacleAgent.psm1

```diff
@@ -1088,6 +1250,14 @@ function Register-Tentacle
         "--console"
     )
 
+    # Get tentacle version to determine spaces compatiblity
+    if (![String]::IsNullOrEmpty($Space))
+    {
+        # Add space name to argument list
+        $argumentList += @("--space", "$Space")
+    }
+
```

Comment:
```suggestion
```

Reply:
Not sure what the suggested change is supposed to be?

Reply:
Deleting the extra whitespace :)